src/components/CategoryScroller/CategoryScroller.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import "./CategoryScroller.css";
+import { v4 as uuid } from "uuid";
+import { useSelectedCategory } from "../../Context/selected-category-context";
+
+const categories = [
+  {
+    topic: "All",
+    id: uuid(),
+    route: "/categories",
+  },
+  {
+    topic: "DSA",
+    id: uuid(),
+    route: "/categories/dsa",
+  },
+  {
+    topic: "DevOps",
+    id: uuid(),
+    route: "/categories/devops",
+  },
+  {
+    topic: "Cloud",
+    id: uuid(),
+    route: "/categories/cloud",
+  },
+  {
+    topic: "React",
+    id: uuid(),
+    route: "/categories/react",
+  },
+  {
+    topic: "JavaScript",
+    id: uuid(),
+    route: "/categories/javascript",
+  },
+  {
+    topic: "CompetitiveProgramming",
+    id: uuid(),
+    route: "/categories/competitive-programming",
+  },
+  {
+    topic: "MachineLearning",
+    id: uuid(),
+    route: "/categories/machine-learning",
+  },
+  {
+    topic: "DataScience",
+    id: uuid(),
```

Comment:
When data is reloaded, this might get change & effect your data updation

Reply:
This id is only used for passing in the key prop of categories, it wouldn't make a diffrence